navigation/HomeNavigator.js

```diff
@@ -1,26 +1,70 @@
-import { Platform } from "react-native";
+import { Platform, TouchableOpacity } from "react-native";
 import { HeaderButtons, Item } from "react-navigation-header-buttons";
-import { createStackNavigator } from "react-navigation-stack";
+import { createStackNavigator, HeaderBackButton } from "react-navigation-stack";
 import CustomHeaderButton from "../components/common/CustomHeaderButton";
-import HomeScreen from "../screens/HomeScreen";
+import PostList from "../screens/PostList";
+import PostDetail from "../screens/PostDetail";
+import PostCreate from "../screens/PostCreate";
 import defaultNavOptions from "./defaultNavOptions";
+import Colors from "../constants/Colors";
+import { FontAwesome } from "@expo/vector-icons";
+
+const leftHeader = (navigation) => (
```

Comment:
variable name should be menuButton or something.
leftHeader is a name for position. content and name should be match.